### DISCUSSION TITLE
My codespaces disappeared along with my files

### DISCUSSION THREAD

**Discussion by ThallyssonKlein** [community/community#3991] [Codespaces] [answered]
I had 2 codespaces with things not saved that I used almost every day. Today I accessed my Github and I simply went back to the waiting list of codespaces and my codespaces disappeared along with my files.

**Comment by 2percentsilk** [comment, marked as answer]
Hey there!
 We're mitigating this issue now. Follow https://github.com/github/feedback/discussions/3992 for updates!
